larpandoracontent/LArObjects/LArMvaInterface.h

```diff
@@ -21,10 +21,66 @@ namespace lar_content
 class MvaTypes
 {
 public:
-    typedef pandora::PandoraInputType<double> MvaFeature;
+    class InitializedDouble
```

Comment:
Add doxygen brief.

Could even call this the MvaFeature class and then remove the typedef below, but this still work for me as is!